apps/fluent-tester/src/TestComponents/Badge/BasicBadgeTest.tsx

```diff
@@ -126,7 +123,7 @@ export const BasicBadge: React.FunctionComponent = () => {
       {svgIconsEnabled && (
         <>
           <Text>Badge with icon</Text>
-          <Badge icon={{ svgSource: svgProps }} iconPosition="after">
+          <Badge icon={{ svgSource: svgProps, ...(Platform.OS === 'android' ? { color: 'gray' } : {}) }} iconPosition="after">
```

Comment:
This should be able to use your new hook, is there a reason you can't?

Reply:
Just a missed replacement on my end 😅